### ISSUE TITLE
what's maxclients=1024?

### ISSUE THREAD

**Issue by AEGEGE** [wheelybird/openvpn-server-ldap-otp#53]
Hello，The following log is prompted during initialization. Does it mean that only a maximum of 1024 clients are supported?

Fri Feb 25 23:36:12 2022 MULTI: multi_init called, r=256 v=256
Fri Feb 25 23:36:12 2022 IFCONFIG POOL: base=11.42.0.4 size=16382, ipv6=0
Fri Feb 25 23:36:12 2022 MULTI: TCP INIT maxclients=1024 maxevents=1028
Fri Feb 25 23:36:12 2022 Initialization Sequence Completed

**Comment by wheelybird** [comment]
Yes.  That's the OpenVPN default.  You can increase this via the OVPN_EXTRA variable if required.

**Comment by AEGEGE** [comment]
Thank you, understood, this is default max-clients of openvpn. 
Can you tell me the OVPN_EXTRA variable writing specification?

**Comment by wheelybird** [comment]
As an example, `OVPN_EXTRA="max-clients = 2046"` should work.

**Comment by AEGEGE** [comment]
Thanks, does this support multi-line configuration? Can you give an example?
For example, I want to enter:
push "route 10.88.x.x 255.255.255.0"
push "route x.232.x.23 255.255.255.255"
max-clients = 2046

**Comment by wheelybird** [comment]
Yes, though you'll need to work out how to pass in a multiline variable to Docker.   However, you can set the routes via `OVPN_ROUTES`.
